pkg/routeagent_driver/handlers/ovn/host_networking.go

```diff
@@ -116,31 +114,60 @@ func (ovn *Handler) programRulesForRemoteSubnets(subnets []string, ruleFunc func
 
 func (ovn *Handler) getNextHopOnK8sMgmtIntf() (*net.IP, error) {
 	link, err := ovn.netLink.LinkByName(OVNK8sMgmntIntfName)
-
 	if err != nil && !errors.Is(err, netlink.LinkNotFoundError{}) {
 		return nil, errors.Wrapf(err, "error retrieving link by name %q", OVNK8sMgmntIntfName)
 	}
 
-	currentRouteList, err := ovn.netLink.RouteList(link, k8snet.IPv4)
-	if err != nil {
-		return nil, errors.Wrapf(err, "error retrieving routes on the link %s", OVNK8sMgmntIntfName)
+	var currentRouteList []netlink.Route
+
+	for _, family := range []k8snet.IPFamily{k8snet.IPv4, k8snet.IPv6} {
```

Comment:
Shouldn't we check the routes only for handler's IP family ?

Reply:
Done.